assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -83,8 +83,15 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 					const { dataTable } = GoogleChart.charts.get( 'user-dimensions-pie-chart' ) || {};
 					if ( dataTable ) {
 						const dimensionValue = dataTable.getValue( row, 0 );
-						setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionValue } );
+
+						if ( 'Others' !== dimensionValue ) {
```

Comment:
@asvinb "Others" is translatable, so the `dimensionValue` value will be different when another locale is selected for the site. We need to translate this `Others` as well to make sure it works for all languages.

Reply:
Updated @eugene-manuilov